package.yaml

```diff
@@ -27,6 +27,9 @@ dependencies:
 - containers
 - reflection
 - constraints
+- random
```

Comment:
I feel like this might be a leftover from the old PR when we also wanted to port `Random` from `polysemy`? But I don't know anything about random-fu so maybe this is a stupid comment and everything is as it should be!

Reply:
Nice catch!  Removed. Good to merge!